components/ResultsContainer.vue

```diff
@@ -82,13 +82,43 @@
         />
       </b-list-group>
     </b-row>
-    <b-col
+    <b-row
       v-if="displayResults"
-      class="d-flex flex-row-reverse"
-      style="margin-top: 1em;"
+      style="margin-top: 1em"
     >
-      <b-row>
-        <div class="download-buttons-div d-flex">
+      <b-col cols="5">
+        <div class="d-flex justify-content-start">
+          <b-button
+            class="nb-button"
+            data-cy="example-usage-button"
+            @click="$modal?.show('example-usage-modal')"
+          >
+            Example usage
```

Comment:
How about
```suggestion
            How to get the data
```
or something like that. Example usage sounds like you mean "... of the query tool"